content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -513,6 +513,18 @@ Your main source of help for troubleshooting your Kubernetes cluster should star
     This was implemented in Kubernetes 1.15, and the pause infrastructure container `mcr.microsoft.com/k8s/core/pause:1.2.0`. Be sure to use these versions or newer ones.
     If you would like to build your own pause infrastructure container, be sure to include [wincat](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/cmd/wincat)
 
+1. My Kubernetes installation is failing because my Windows Server node is behind a proxy 
+  
+    If you are behind a proxy, the following PowerShell environment variables must be defined:
+    ```PowerShell
+    [Environment]::SetEnvironmentVariable("HTTP_PROXY", "http://proxy.example.com:80/", [EnvironmentVariableTarget]::Machine)
+    [Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
+    ```
+
+1. What is a `pause` container?
```

Comment:
This should include more details on (or link to) why a pause container is needed in kubernetes

Reply:
Yes, I actually didn't write this originally just moved it to the right section after rebasing but I added an explanation now, thanks!